packages/sit-onyx/src/components/OnyxCheckbox/OnyxCheckbox.ct.tsx

```diff
@@ -213,3 +214,23 @@ for (const testCase of invalidTestCases) {
     await expect(component).toHaveScreenshot(`invalid-${testCase.name}.png`);
   });
 }
+
+const STATES = {
+  state: ["default", "disabled", "invalid", "required", "optional"],
```

Comment:
Invalid screenshots does not seem to work, they should be read.
Invalid only triggers when the checkbox is touched so it needs to be blurred first